src/SparkFun_External_EEPROM.h

```diff
@@ -111,6 +111,8 @@ struct struct_memorySettings
     uint8_t writeTime_ms;
     bool pollForWriteComplete;
     uint8_t addressSize_bytes;
+    bool usingWP;
```

Comment:
Rather than use a bool, I recommend setting the wpPin to 255 and then test against it: if(settings.wpPin != 255) digitalWrite(...

Reply:
If a user sets the pin to something other than 255, WP control is enabled.